packages/trpc/server/routers/viewer/admin/setSMSLockState.handler.ts

```diff
@@ -16,6 +16,20 @@ type GetOptions = {
 const setSMSLockState = async ({ input }: GetOptions) => {
```

Comment:
This mutation is only called inside `apps/web/pages/settings/admin/lockedSMS/lockedSMSView.tsx` where you can mark the user or team as LOCKED or UNLOCKED only. This aren't the changes required.